guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -17,6 +16,31 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v2002.0.22
+
+-  {:.new}**Changes to the ece-tools composer package**
+
+   We changed the structure of the {{ site.data.var.ct }} package to reduce complexity for scheduling updates and working with community contributions.
+
+   -  {:.new}{{ site.data.var.ece }} patches are no longer included in the {{ site.data.var.ct }} ece-tools package to allow Magento to manage and deliver patches independently without requiring an {{ site.data.var.ct }} package release. These patches are now delivered in a separate Magento Cloud Patches module that is a required dependency for the {{ site.data.var.ct }} package.
+
+   -  {:.new}Updated the composer requirements for the {{ site.data.var.ct }} package to add a dependency for the Magento Cloud Patches module `magento/magento-cloud-patches`.
+
+-  {:.new}**Magento Cloud Patches package**–Initial release of the Magento Cloud Patches module which delivers Magento-provided patches and critical fixes in a separate composer package. Module source files are available in the [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) GitHub repository.
+
+   Magento Cloud Patches, version 1.0 adds the following patches for {{ site.data.var.ee }} applications hosted on the Cloud platform.
+
+   -  {:.new}<!--MAGECLOUD-4649-->**Page Builder security patches for 2.3.1.x and 2.3.2.x releases**–Fixes an issue in Page Builder preview that allows unauthenticated users to access some templating methods that can be used to trigger arbitrary code execution over the network (RCE) resulting in global information leaks. This issue can occur when using unsupported versions of Page Builder, such as Page Builder Beta with {{ site.data.var.ee }} versions 2.3.1 and v2.3.2.
+
+   -  {:.new}<!--MAGECLOUD-4428-->**MSI patches**–Fixes issues that caused indexing errors and performance issues when using default inventory settings for managing stock.
+
+   -  {:.new}<!--MAGECLOUD-4422-->**Backward Compatibility of new Mail Interfaces**-Fixes a backward incompatibility issue caused by the new `Magento\Framework\Mail\EmailMessageInterface` introduced in {{ site.data.var.ee }} v2.3.3. In the scope of this patch, the new `EmailMessageInterface` inherits from the old `MessageInterface`, and {{ site.data.var.ee }} core modules are reverted to depend on `MessageInterface`.
+
+   -  {:.new}<!--MAGECLOUD-4448-->**Catalog pagination does not work on Elasticsearch 6.x**–Fixes a critical issue with search result pagination that affects customers using Elasticsearch 6.x as the catalog search engine.
+
+  {:.bs-callout-warning}
+  You must update to {{ site.data.var.ct }} version 2002.0.22 or later to apply the patches included in Magento Cloud Patches, version 1.0.
```

Comment:
Do we need to create a variable for the package? `mcp: Magento Cloud Patches`

Reply:
I have created variables for all the components, but I'm going to add them in the 2002.1.0 release.